<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inquiry</class>
 <widget class="QMainWindow" name="Inquiry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pricer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Note">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>输入饰品名称（查询多个时用&quot;;&quot;隔开）：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>650</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sure">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>精确查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Names">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Warning">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Result">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>550</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Fuzzy">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>模糊查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Up_sort">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>价格升序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Down_sort">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>价格降序</string>
    </property>
   </widget>
   <widget class="QLabel" name="Result_sum">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="More_results">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加载更多</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
